astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,51 @@
+
+![Napster app from Wikipedia](/img/blogs/i-just-want-authentication-to-work/napster.png)
+
+## The benefits of using an authentication service
+
+There are many benefits to using an authentication service. Authentication services can help you to:
+
+* Save time and money: Implementing and maintaining authentication can be time-consuming and expensive. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Improve security: Authentication services can help you to improve the security of your applications by using best practices and the latest security technologies.
+* Reduce the risk of compliance violations: Authentication services can help you to meet compliance requirements by providing features such as two-factor authentication and user provisioning.
+
+The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house and didn't secure!
+
+![Meme you can't leak passwords if you don't store passwords, man pointing at head](/img/blogs/i-just-want-authentication-to-work/meme-leak.png)
+
+## Choosing an authentication service
+
+When choosing an authentication service, there are a few factors to consider:
+
+* Features: Consider the features that are important to you, such as two-factor authentication, user provisioning, and support for different authentication methods.
+* Pricing: Authentication services can vary in price. Compare the pricing of different services to find the best deal for your needs.
+* Ease of use: Consider how easy the authentication service is to use. You want a service that is easy to set up and manage.
+* Hosting: Do you need flexibility to host your own instance of an Authentication Provider or have them Host.
+
+## For developers who don't care about implementing authentication
+
+If you are a developer who doesn't care about implementing authentication, there are a few things that you can do to make your life easier:
+
+* Use an authentication service. There are many different authentication services available, both free and paid. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Use a framework or library that provides authentication functionality. There are many different frameworks and libraries available that provide authentication functionality. Using a framework or library can save you the time and effort of having to implement authentication yourself. If you are ready to get started checkout our [quickstarts](/docs/quickstarts/).
```

Comment:
may want to say FusionAuth Quickstarts.

I know you don't want it to be "salesy", but you haven't mentioned fusion auth at all to this point so "our quickstars" might not be clear.